src/server/js/resources/houses/index.js

```diff
@@ -1,6 +1,6 @@
 import express from 'express';
 
-import House from './model';
+import HouseModel from './model';
```

Comment:
i've gone back and forth on this one a bit... the mongoose models will be import as `XyzModel`